ocis-pkg/service/grpc/keepalive.go

```diff
@@ -0,0 +1,24 @@
+package grpc
+
+import (
+	"math"
+	"os"
+	"time"
+)
+
+const (
+	_serverMaxConnectionAgeEnv = "GRPC_MAX_CONNECTION_AGE"
+
+	// same default as grpc
+	infinity                 = time.Duration(math.MaxInt64)
```

Comment:
LOL. `math.MaxInt64` is NOT infinity because `math.MaxInt64 + 1` is bigger.

Reply:
LOL, nerdy but ur correct